js/piwik.js

```diff
@@ -2516,6 +2582,7 @@ if (typeof Piwik !== 'object') {
              * Send request
              */
             function sendRequest(request, delay, callback) {
+                lastTrackerRequestTime = (new Date()).getTime();
```

Comment:
Also I don't know if we have to update this variable when `sendBulkRequest()` is called